lib/libc/string/wcscasecmp.c

```diff
@@ -41,7 +41,7 @@ wcscasecmp(const wchar_t *s1, const wchar_t *s2)
 		c1 = towlower(*s1);
 		c2 = towlower(*s2);
 		if (c1 != c2)
-			return ((int)c1 - c2);
+			return (int)((unsigned int)c1 - (unsigned int)c2);
```

Comment:
style(9) requires parentheses around the entire expression including the cast, though IDK if we prefer the explicit cast like this or just let it be implicitly cast on return